src/main/java/com/zendesk/maxwell/schema/SchemaStore.java

```diff
@@ -276,11 +392,16 @@ private void restoreSchemaMetadata(Connection conn, Long schemaID) throws SQLExc
 		LOGGER.info("Restoring schema id " + schemaRS.getInt("id") + " (last modified at " + this.position + ")");
 
 		this.schema_id = schemaRS.getLong("id");
+		this.base_schema_id = schemaRS.getLong("base_schema_id");
+
+		if ( schemaRS.wasNull() )
```

Comment:
It'd be `0`.  For some unbelievable reason.  http://stackoverflow.com/questions/2920364/checking-for-a-null-int-value-from-a-java-resultset